src/lib.rs

```diff
@@ -1539,6 +1539,29 @@ impl Node {
 		self.payment_store.remove(&payment_id)
 	}
 
+	/// Alby: Return encoded channel monitors for a recovery of last resort
+	pub fn get_encoded_channel_monitors(&self) -> Result<Vec<KeyValue>, Error> {
+		let channel_monitor_store = Arc::clone(&self.kv_store);
```

Comment:
Nitpick: why clone these?..

Reply:
I am not sure why it is needed, but every other place it's done in this way, so am just following convention

Reply:
They do it when spawning tasks, which makes sense: the tasks receive the cloned handle ;)